yoti-sdk-api/src/test/java/com/yoti/api/client/spi/remote/call/ReceiptOutcome.java

```diff
@@ -0,0 +1,9 @@
+package com.yoti.api.client.spi.remote.call;
+
+public enum ReceiptOutcome {
+    SUCCESS(true),
+    FAILURE(false);
+
+    ReceiptOutcome(boolean successful) { }
```

Comment:
what's the point of having the vars if they won't be set or accessed? Why not just:
```java
public enum ReceiptOutcome {
    
    SUCCESS,
    FAILURE;

}
```